documentation/docs/parent-pages/exercises.md

```diff
@@ -0,0 +1,18 @@
+# Testnet Exercises
+-----
+
+<!-- panels:start -->
+
+<!-- div:right-panel -->
+   <center>
+  <a href="../#/Exercises/exercise-4">
+<img src="https://i.ibb.co/dBGGTFR/cropped.jpg"
```

Comment:
Not sure whether we prefer to host all assets locally.  I guess it's okay for now.

Reply:
ahh makes sense - I got a little lazy there 😛 

From what I have observed I think it caches some stuff in local browser cache too (not really sure though) - so these assets will be fetch once and then would be easily loaded for every click after that.

But yeah, feel free to ask me - I can definitely move all the hosted images to a local `_media` or `_ assets` folder  in this or any of the following PRs

Reply:
okay so this is done - all hosted images should be now in `_media` directory